twilio/base/version.py

```diff
@@ -55,10 +55,13 @@ def exception(cls, method, uri, response, message):
         # noinspection PyBroadException
         try:
             error_payload = json.loads(response.text)
+            details = {}
             if 'message' in error_payload:
                 message = '{}: {}'.format(message, error_payload['message'])
+            if 'details' in error_payload:
+                details = error_payload['details']
```

Comment:
nit: Can reduce these 3 added lines to just `details = error_payload.get('details')`

Reply:
fixed, thanks !